build.gradle

```diff
@@ -23,8 +23,3 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
-task wrapper(type: Wrapper) {
```

Comment:
The new syntax caused a flatout error when trying to use the project as a submodule. My guess is that only the root module gets a wrapper task. Since all of the information is already encoded in `gradle/wrapper/gradle-wrapper.properties`' `distributionUrl` anyway, I just threw that entire part out. I will, of course, put it back in if that's your wish.

Reply:
How about `project.tasks.findByName('wrapper')?.configure {`?

Reply:
Ok I didn't know it could cause a problem - you can leave it out then.